app/forms/cohorts/bulk_import_students_form.rb

```diff
@@ -31,10 +31,21 @@ def save
     def csv_rows
       @csv_rows ||=
         begin
-          CSV.read(csv, headers: true).map { |r| r.to_hash }
+          CSV.read(csv, headers: true, col_sep: col_sep(csv) || ',').map { |r| r.to_hash }
```

Comment:
Consider letting `col_sep` return `,` by default to eliminate the use of `||` operator.